collections/_drafts/agent-forwarding.md

```diff
@@ -20,8 +20,8 @@ services. With SSH keys, you can connect to GitHub without supplying your
 username and personal access token at each visit.
 
 When you set up SSH, you need to generate a new SSH key and add it to the
-`ssh\-agent`. You must add the SSH key to your account on GitHub before you
-use the key to authenticate.
+`ssh\-agent`. You must add the SSH key to your account on GitHub before you use
```

Comment:
yeah, we need to come up w/ some clear way to consistently refer to this w/o _always_ being **code**

```suggestion
`ssh-agent`. You must add the SSH key to your account on GitHub before you use
```

Reply:
> some clear way to consistently refer to this w/o always being code

pls check out the comment in #1111 i have linked below:

🔗 https://github.com/OpenINF/openinf.github.io/issues/1111#issuecomment-1922154764

Reply:
![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/a9656f39-06db-4ef1-9b9c-35850d97a363)


alright, according to that, a "text-mode command name" should result in _bold red text in a monospaced font_

Reply:
```suggestion
***[`ssh-agent`][]***. You must add the SSH key to your account on GitHub before you use
```